dragon-scales/src/app/components/StepViewerComponents/StepViewer.jsx

```diff
@@ -47,107 +56,153 @@ const StyledStepPaperActive = styled(Paper)(({ theme }) => ({
     paddingLeft: theme.spacing(2),
     paddingTop: theme.spacing(1),
     paddingBottom: theme.spacing(1),
+}));
 
-}))
-
-const StyledStepTittleTypography = styled(Typography)(({ theme }) => ({
+const StyledStepTitleTypography = styled(Typography)(({ theme }) => ({
     fontWeight: 'bold',
     fontSize: theme.typography.h6.fontSize,
-}))
+}));
 
 const StyledStepContentBlocksTypography = styled(Typography)(({ theme }) => ({
     fontSize: theme.typography.body1.fontSize,
-}))
+}));
 
 const StyledNewContentBox = styled(Box)(({ theme }) => ({
     display: "flex",
     alignItems: "center",
-    paddingTop: "10px"
-
-}))
-
-
-
-
-export default function StepViewer( { stepEntity, markStepState, reloadTask } ) {
+    paddingTop: "10px",
+}));
 
+function StepViewer({ 
+    stepEntity, 
+    markStepState = () => {}, 
+    reloadTask = () => {}, 
+    parentID = '', 
+    parentName = '', 
+}) {
```

Comment:
why are you changing the signature here? None of these arguments should have defaults, they need a reload task and markStepState argument there.